### ISSUE TITLE
Properly implement and support color bars being served separately from the map

### ISSUE THREAD

**Issue by joernu76** [Open-MSS/MSS#1375]
The WMS standard supports delivering color bars as separate image. This is supported in a very basic fashion by the MSUI, but not at all by the MSWMS. In particular in combination with multi-layering, the colorbar must be displayed separately to remain usable (currently all our colorbars are displayed at the same location).

This issue entails
1) implementing support for this feature in the MSWMS backend
2) properly address "nice" displaying of the colorbar in the frontend for typical use-cases (i.e. less than 3 bars)
3) Revamping the existing plotting layers to use this feature.

**Comment by joernu76** [comment]
@ReimarBauer 
do we have a notion of marking something as GSOC issue? I am not sure this can still be added to 2021?

**Comment by ayeankit** [comment]
Hello @joernu76 , Is this still open?

**Comment by ReimarBauer** [comment]
@ayeankit Yes this belongs to one of our ideas for a task during gsoc https://open-mss.github.io/develop/GSOC/color_bars_served
